sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -799,6 +805,17 @@ def run_subprocess():
         logging.warning("Could not decode %s data due to %s", test_name, e)
         continue
 
+      if test_name == "named_tuple_simple" and not is_using_dill:
+        should_have_relative_path = not compat_version
```

Comment:
could you add a comment explaining why `should_have_relative_path = not compat_version`